### ISSUE TITLE
version 1.2.2 autotrack doesn't make MCIR rain image

### ISSUE THREAD

**Issue by svScoots** [SatDump/SatDump#882]
I have been using Satdump 1.2.1 with autotrack to capture NOAA APT and Meteor images.  I updated to version 1.2.2 compiled on my machine from source (as I did with 1.2.1) The MCIR-rain view is no longer being made correctly as it was for 1.2.1 The first image is all black and about 10K the MCIR with map image only shows the map on a black background.

Raspberry Pi 4

1.2.2 released version

the log shows the image was made and shows the map used from cache to make the second one, there were no obvious errors in the log.

[20:48:52 - 01/12/2024] (T) Max Total Used  909
[20:48:52 - 01/12/2024] (T) Offset for cha is 0
[20:48:52 - 01/12/2024] (T) Offset for chb is 0
[20:48:52 - 01/12/2024] (D) Filtering timestamps...
[20:49:05 - 01/12/2024] (I) Saving /home/share/wxsat/img/2024-12-01_20-28_noaa_apt_137.912 MHz/./avhrr_3_rgb_MCIR_Rain_(Uncalibrated).png...
[20:49:06 - 01/12/2024] (I) Applying cached map overlay...
[20:49:06 - 01/12/2024] (I) Saving /home/share/wxsat/img/2024-12-01_20-28_noaa_apt_137.912 MHz/./avhrr_3_rgb_MCIR_Rain_(Uncalibrated)_map.png...
[21:01:52 - 01/12/2024] (T) Shutting down FFT
[21:46:42 - 01/12/2024] (W) Signal Received. Stopping.
[21:46:42 - 01/12/2024] (I) Done! Goodbye

**Comment by svScoots** [comment]
I get MCIR as normal too. I don't get the MCIR-rain (the colored version) which I thought used the same IR data.
I've pasted the whole log below.

[savesatdump-20241201T211304.log](https://github.com/user-attachments/files/17983021/savesatdump-20241201T211304.log)

**Comment by svScoots** [comment]
Well, that worked, I now have MCIR-rain charts again, thanks much.

It is somewhat un-satisfying as I did not find any loose remaining files.  I did a complete uninstall, manually checked the places you suggested.  Didn't find any files, only empty directories, although I admit I didn't follow down each directory tree, I deleted some en masse. I did look through what I thought were the likely ones.  I then re-checked all the dependencies, but none were missing and recompiled and reinstalled. All works as before the update to 1.2.2. Thanks.
